server/src/main/kotlin/org/web3j/openapi/server/config/ContractAddresses.kt

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright 2020 Web3 Labs Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package org.web3j.openapi.server.config
+
+import org.web3j.abi.datatypes.Address
+
+data class ContractAddresses(
```

Comment:
Are you sure we need an extra class? 

I think we could add the addresses map directly on the Jersey properties.

Or, if you need for the HK2 binder, I think should be better to inherit from `HashMap<String, Address>`.

Reply:
like this ?